hello-jniCallback/app/build.gradle

```diff
@@ -11,25 +11,33 @@ android {
         versionName "1.0"
         externalNativeBuild {
             cmake {
-                abiFilters 'x86', 'x86_64', 'armeabi', 'armeabi-v7a', 'arm64-v8a'
+                arguments '-DANDROID_PLATFORM=android-9',
+                    '-DANDROID_TOOLCHAIN=clang', '-DANDROID_STL=gnustl_static'
             }
         }
     }
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android.txt'),
+                          'proguard-rules.pro'
         }
     }
     externalNativeBuild {
         cmake {
             path "src/main/cpp/CMakeLists.txt"
         }
     }
+    productFlavors {
+        combined {
```

Comment:
yes, put them back to defaultConfig for all samples building one productFlavor: since assemble and assembleCombined are exactly the same.  thanks